### PR TITLE
feat: added story reaction

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/hooks/useStory.ts [hunk 1]
```diff
@@ -26,8 +30,41 @@ export const useStory = ({ targetId, targetType }: IUserStory) => {
...
+          await ReactionRepository.removeReaction(
+            'story',
+            targetId,
+            reactionName
+          );
+        } else {
+          await ReactionRepository.addReaction('story', targetId, reactionName);
+        }
+      } catch (error) {
+        Alert.alert('Error on react');
```
Comment (chaiwattsw) [inline src/hooks/useStory.ts:57]: maybe better if we show alert with some useful infomation that throw from sdk if have any error

Comment (frankent) [reply src/hooks/useStory.ts:57]: Agree with @chaiwattsw

Comment (NaingAmity) [reply src/hooks/useStory.ts]: Added sdk error in Alert krub

**Review by bonanaaaaaa — APPROVED** [review]
LGTM

**Pull request merged**